### DISCUSSION TITLE
Please tell me your favorite online-game (or social-game) :)!

### DISCUSSION THREAD

**Discussion by IppeiOTAKE** [NAIST-SE/AttractingContributors#452] [General]
Please tell me your favorite online-game (or social-game)!

My fork: https://github.com/IppeiOTAKE/AttractingContributors-OnlineGame-or-SocialGame

I like "Uma Musume: Pretty Derby" now :) (and my favorite charactor is "Yayoi Akikawa")

**Comment by Kotjik** [comment]
Hey, I contributed to your repo - would you mind adding something to mine as well? 🙂
Here's my repo: https://github.com/Kotjik/AttractingContributors-FavoriteCharacter

**Reply by IppeiOTAKE** [reply]
Thank you very much, I just checked and it looks like already voted (Akigawa is by me).
And I just confirmed that you MERGE it, thank you!
